src/app.jsx

```diff
@@ -2,16 +2,19 @@ const React = require('react');
 const ReactDOM = require('react-dom');
 const YearBook = require('./components/YearBook');
 const PersonShow = require('./components/PersonShow');
+const Router = require('react-router').Router
+const Route = require('react-router').Route
+const Link = require('react-router').Link
+const browserHistory = require('react-router').browserHistory;
+
 
 const App = React.createClass({
   render: function() {
     return (
-      <div>
-        <h1>Yearbook Component</h1>
-        <YearBook />
-        <h1>PersonShow Component (Hardcoded to Person 0)</h1>
-        <PersonShow />
-      </div>
+      <Router history={browserHistory}>
+        <Route path="/" component={YearBook} />
+        <Route path="/person/:idx" component={PersonShow} />
+      </Router>
```

Comment:
Nice job setting up routes. If you go back to this assignment later, make sure to add the `About` route too